ice40/devices/top-routing-virt/CMakeLists.txt

```diff
@@ -32,7 +32,9 @@ define_device_type(
   ARCH ice40
   ARCH_XML arch.xml
   SCRIPT_OUTPUT_NAME timing
+  SCRIPT_DEPS ${SDF_FILE}
```

Comment:
This needs to be a call to append_file_depedencies, etc.

Reply:
Done.